src/GitVersionCore/Core/RepositoryMetadataProvider.cs

```diff
@@ -294,24 +296,24 @@ public BranchCommit FindCommitBranchWasBranchedFrom(IBranch branch, Config confi
                 throw new ArgumentNullException(nameof(branch));
             }
 
-            using (log.IndentLog($"Finding branch source of '{branch.FriendlyName}'"))
+            using (log.IndentLog($"Finding branch source of '{branch.Name.FriendlyName}'"))
```

Comment:
Perhaps we can make `Branch.ToString()` return `Name.ToString()` which again returns `FriendlyName` so we can simplify it down to `branch` every time we want its `string` representation?

```suggestion
            using (log.IndentLog($"Finding branch source of '{branch}'"))
```

Reply:
Done where it made sense